### ISSUE TITLE
Pip Installation Error

### ISSUE THREAD

**Issue by davidpgil** [ihabunek/toot#18]
I just installed pip, and the dependencies in requirements.txt and requirements-dev.txt. I typed "pip install toot" and I get an error message. Please advise.

```
Downloading/unpacking toot
  Downloading toot-0.12.0.tar.gz
  Running setup.py egg_info for package toot
    Traceback (most recent call last):
      File "<string>", line 16, in <module>
      File "c:\users\johnDoe\appdata\local\temp\pip_build_johnDoe\toot\setup.py", line 8, in <module>
        with open("requirements.txt") as f:
    IOError: [Errno 2] No such file or directory: 'requirements.txt'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 16, in <module>

  File "c:\users\johnDoe\appdata\local\temp\pip_build_johnDoe\toot\setup.py", line 8, in <module>

    with open("requirements.txt") as f:

IOError: [Errno 2] No such file or directory: 'requirements.txt'

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in c:\users\johnDoe\appdata\local\temp\pip_build_johnDoe\toot
Storing complete log in C:\Users\johnDoe\pip\pip.log
```

**Comment by FRidh** [comment]
@ihabunek the sdist that was uploaded to PyPI is incomplete.

@davidpgil for now you could install directly from the git repository instead.

**Comment by ihabunek** [comment]
@davidpgil sorry for the long wait. This should not happen with 0.14.0+, i have removed access to requirements.txt from setup.py. Please try it out.

**Comment by davidpgil** [comment]
Its seems the installation now works! However, typing "toot" throws the error 

```
Traceback (most recent call last):
  File "C:\Python27\Scripts\toot-script.py", line 9, in <module>
    load_entry_point('toot==0.14.0', 'console_scripts', 'toot')()
  File "C:\Python27\lib\site-packages\pkg_resources.py", line 357, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "C:\Python27\lib\site-packages\pkg_resources.py", line 2394, in load_entry_point
    return ep.load()
  File "C:\Python27\lib\site-packages\pkg_resources.py", line 2108, in load
    entry = __import__(self.module_name, globals(),globals(), ['__name__'])
  File "C:\Python27\lib\site-packages\toot\console.py", line 11, in <module>
    from toot import config, api, commands, ConsoleError, CLIENT_NAME, CLIENT_WEBSITE
  File "C:\Python27\lib\site-packages\toot\commands.py", line 19, in <module>
    from toot.app import TimelineApp
  File "C:\Python27\lib\site-packages\toot\app.py", line 5, in <module>
    import curses
  File "C:\Python27\lib\curses\__init__.py", line 15, in <module>
    from _curses import *
ImportError: No module named _curses
```

**Comment by ihabunek** [comment]
I think curses is not supported on windows so it breaks the whole program. I will try to make the commands which don't need curses work on windows.

**Comment by ihabunek** [comment]
I have released v0.15 which should address this issue. You should be able to use all commands except `toot curses`. Upgrade and try it out.

```
pip install --upgrade toot
```
